hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestContainerHealthTask.java

```diff
@@ -299,7 +304,7 @@ public void testDeletedContainer() throws Exception {
         .thenReturn(new ContainerWithPipeline(mockContainers.get(0), null));
 
     List<UnhealthyContainers> all = unHealthyContainersTableHandle.findAll();
-    assertThat(all).isEmpty();
+    assertTrue(all.isEmpty());
```

Comment:
Please keep `assertThat`, added by HDDS-10034.

```suggestion
    assertThat(all).isEmpty();
```